.nvmrc

```diff
@@ -1 +1 @@
-7.*
```

Comment:
Might be worth sending out an email about this before landing.  Probably not a huge deal, though.